sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -242,74 +241,59 @@ def _microsoft_teams_user_raw_id(identifier: MicrosoftTeamsUserIdentifier) -> st
     return f"{TEAMS_USER_PUBLIC_CLOUD_PREFIX}{user_id}"
 
 
-_MicrosoftBotProperties = TypedDict(
-    "MicrosoftBotProperties",
+MicrosoftTeamsAppProperties = TypedDict(
```

Comment:
We may need to keep the old ones around.

Reply:
Reverted the deletions.